tests/test_ml.py

```diff
@@ -86,6 +94,37 @@ def test_output_values_missing(self):
             )
         )
 
+    def test_tree_predict(self):
```

Comment:
@jgrss  is there any way to get ci to run the test_ml.py only for the geowombat-ml add on?  I don't want to pin scikit-learn=1.2.0 for everything unless we need to.

Reply:
Maybe here? https://github.com/jgrss/geowombat/blob/28c765a73d378b7e9ce0ab8cbfc28b8dcdb44153/.circleci/config.yml#L71